app/services/my_anime_list_sync_service.rb

```diff
@@ -11,36 +11,48 @@ def initialize(library_entry, method)
 
   def execute_method
     # TODO: add errors later on if we can't find mal data
-    return if mal_media.nil?
-
     case method
     when 'delete'
-      delete("#{media_type}list/#{media_type}/#{mal_media_id}", linked_profile)
+      media_type_d = library_entry['media_type'].underscore
```

Comment:
Allow you to? Yes. Should you do it? No.
For reference: [check this](http://stackoverflow.com/questions/2985850/how-does-ruby-allow-a-method-and-a-class-with-the-same-name/2986338#2986338)